src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -162,7 +162,6 @@ public String getJsonString() {
     
     @Override
     public void sanitizeForSaving() {
-        this.feedbackSessionName = Sanitizer.sanitizeTitle(feedbackSessionName);
```

Comment:
@kanghj thanks, please take over this PR for me. I has been quite busy in these 2 months 😣